ui/tests/integration/components/pki/pki-key-form-test.js

```diff
@@ -138,35 +138,32 @@ module('Integration | Component | pki key form', function (hooks) {
     await click(SELECTORS.keyCancelButton);
     assert.true(this.model.isDestroyed, 'new model is unloaded on cancel');
 
-    /* COMMENT IN WHEN EDIT IS COMPLETE
-      this.store.pushPayload('pki/key', {
-      modelName: this.modelName,
-      keyName: 'test-key',
+    this.store.pushPayload('pki/key', {
+      modelName: 'pki/key',
+      key_name: 'test-key',
       type: 'exported',
-      keyId: 'some-key-id',
-      keyType: 'rsa',
-      keyBits: '2048',
-    };);
-      this.model = this.store.peekRecord('pki/key', this.data.keyId;
-
-      await render(
-        hbs`
+      key_id: 'some-key-id',
+      key_type: 'rsa',
+      key_bits: '2048',
+    });
+    this.model = this.store.peekRecord('pki/key', 'some-key-id');
+
+    await render(
+      hbs`
           <PkiKeyForm
             @model={{this.model}}
             @onCancel={{this.onCancel}}
             @onSave={{this.onSave}}
           />
         `,
-        { owner: this.engine }
-      );
+      { owner: this.engine }
+    );
 
-      await fillIn(SELECTORS.keyNameInput, 'new-name');
-      await click(SELECTORS.keyCancelButton);
-      assert.strictEqual(this.model.keyName, undefined, 'Model attributes rolled back on cancel');
-  */
+    await fillIn(SELECTORS.keyNameInput, 'new-name');
+    await click(SELECTORS.keyCancelButton);
+    assert.strictEqual(this.model.keyName, 'test-key', 'Model attributes rolled back on cancel');
```

Comment:
We should assert that the keyName updated on input before we cancel, to verify it does change the model as expected

Reply:
It actually doesn't do this right now...which I discovered writing this test a couple weeks ago - I have a PR to fix.